build.sbt

```diff
@@ -395,14 +439,18 @@ lazy val scalalib =
     .in(file("scalalib"))
     .enablePlugins(MyScalaNativePlugin)
     .settings(
-      // This build uses libScalaVersion, which is currently 2.11.12
-      // to compile what appears to be 2.11.0 sources. This yields 114
+      // This build uses Scala 2.11 version 2.11.12 to compile
```

Comment:
"Scala 2.11 version 2.11.12 " Eh?  Noticed whilst looking for changes test framework,
agreed a nit.

Suggested alternatives,   "Scala 2.11.12" or "Scala 2.11, version 2.11.12, "